_data/nav.yml

```diff
@@ -0,0 +1,183 @@
+primary:
```

Comment:
@dawnpm @arantxarecio this "primary" object is what holds the information for the side navigation. The nested children will only appear when on a page in the parent folder (ex. https://cg-a72b65a7-73f0-4859-a1d7-dc953cf6ade8.sites.pages.cloud.gov/preview/gsa/search-gov-website/sidenav-updates/indexing/) 

Ahead of formal PR review, feel free to provide any feedback on what items we want under each section. The support section got a bit weird because the links are not within that subfolder, so I removed those for now.